src/unstract/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -81,7 +81,7 @@
       "type": "string",
       "title": "Page number(s) or range to extract",
       "default": "",
-       "pattern": "^[\\d\\s]+[\\d\\-,\\s]*",
+      "pattern": "^(\\s|\\d+)[\\d\\-,\\s]*",
```

Comment:
Did we test with all cases here? 
We can check/test like  `match = re.match(pattern, test)`

Reply:
I have tested with the usual suspects. Hope that helps @muhammad-ali-e

Reply:
Used https://regex101.com/ for testing